cdap-api/src/main/java/co/cask/cdap/api/RuntimeContext.java

```diff
@@ -50,4 +51,11 @@
    * @return an {@link Admin} to perform admin operations.
    */
   Admin getAdmin();
+
+  /**
+   * @param tracerName the name of the logger using which the debug information will be logged
+   *
+   * @return an {@link DataTracer} to perform log operations.
+   */
+  DataTracer getTracer(String tracerName);
```

Comment:
I think `getDataTracer` would be more correct name since its called on `RuntimeContext`